cli/fmt.rs

```diff
@@ -165,14 +165,14 @@ fn get_matching_files(glob_paths: Vec<String>) -> Vec<PathBuf> {
 /// First argument supports globs, and if it is `None`
 /// then the current directory is recursively walked.
 pub fn format_files(
-  maybe_files: Option<Vec<String>>,
+  maybe_files: Option<Vec<PathBuf>>,
   check: bool,
 ) -> Result<(), ErrBox> {
   // TODO: improve glob to look for tsx?/jsx? files only
-  let glob_paths = maybe_files.unwrap_or_else(|| vec!["**/*".to_string()]);
+  let glob_paths = maybe_files.unwrap_or_else(|| vec![PathBuf::from("**/*")]);
```

Comment:
This seems odd/wrong `PathBuf::from("**/*")` ...

Reply:
I agree with @ry - it'd be better to rename field on `Subcommand::Format` to `globs`. 

Otherwise +1 from me, stricter types are always good

Reply:
As far as I've seen it's normal for glob patterns to be stored as "paths". They have distinctly separated segments (since any special syntax is limited to a segment), absoluteness, parent paths, it's useful to `join()` them, `normalize()` them, etc.  Are you sure this is a problem?

Reply:
Yeah, makes sense, I've seen `glob`/`globset` crate do that. LGTM